packages/ds-healthcare-gov/docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -107,7 +121,21 @@
         </div>
       </div>
       <span class="ds-c-field__error-message ds-c-field__error-message--inverse" role="alert">
-        Please enter a year in the past
+        <svg
+          aria-labelledby="icon-2__title"
+          aria-hidden="true"
+          class="ds-c-icon ds-c-icon--alert-circle"
+          focusable="false"
+          id="icon-2"
+          role="img"
+          viewBox="36 -12 186 186"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <title id="icon-2__title">Alert</title>
```

Comment:
there are two elements on this page that have `id="icon-2__title"` consider updating one so that no two elements on the page have the same id. Will also need to update line 125 as part of this change

Reply:
Same situation for the form-label & textfield examples